--- mms-1.1.0-rc9/regextools.cpp.orig	2009-12-30 10:13:09.000000000 +0000
+++ mms-1.1.0-rc9/regextools.cpp	2009-12-30 10:13:26.000000000 +0000
@@ -1,6 +1,7 @@
 #include "regextools.h"
 #include <cassert>
 #include <pcrecpp.h>
+#include <stdio.h>
 using std::string;
 using std::vector;
 using std::pair;
--- mms-1.1.0-rc9/plugins/feature/movie/Movie_info.cpp.orig	2009-12-30 10:42:20.000000000 +0000
+++ mms-1.1.0-rc9/plugins/feature/movie/Movie_info.cpp	2009-12-30 10:42:41.000000000 +0000
@@ -24,6 +24,7 @@
 #include <math.h>
 #include <stdarg.h>
 #include <errno.h>             /* per  errno */
+#include <stdio.h>
 #include "endianesstools.h"
 #include "ftyp.h"
 //#define __func__  __PRETTY_FUNCTION__
